src/lib.rs

```diff
@@ -78,14 +78,12 @@
 
 #[macro_use]
 extern crate log;
-extern crate serde;
 #[macro_use]
 extern crate serde_derive;
```

Comment:
You could import the macros directly where needed instead of using `#[macro_use]`:

    use serde::{Serialize, Deserialize};

Reply:
Fixed.